doc/fuzzing.md

```diff
@@ -0,0 +1,62 @@
+# Fuzzing
+
+## Setting up a fuzzer locally
+
+As of v1.8, OpenConsole can now be built with a `Fuzzing` configuration.

```

Comment:
Probably don't say anything about the version. It's definitely not 1.8 anymore...

Reply:
yup.